build/markdown.m4

```diff
@@ -49,13 +49,13 @@ AC_DEFUN([GP_CHECK_MARKDOWN],
     GTK_VERSION=2.16
     WEBKIT_VERSION=1.1.13
 
-    GP_CHECK_GTK3([webkit_package=webkitgtk-3.0],
+    GP_CHECK_GTK3([webkit_package=webkit2gtk-4.0],
```

Comment:
@hyperair I wonder if we can cascade from `webkit-1.0` to `webkit-3.0` to `webkitgtk-4.0` to try and support all versions?

Reply:
Not tested, but I mean like this:

```sh
GP_CHECK_GTK3([webkit_package=webkit-3.0], [
  GP_CHECK_GTK3([webkit_package=webkit2gtk-4.0], [
    webkit_package=webkit-1.0
  ]
])
...
AM_CONDITIONAL([MARKDOWN_WEBKIT2], [test "x$webkit_package" = "xwebkit2gtk-4.0"])
...
```

@b4n @sardemff7 @dmaphy @evgeni @hyperair care to comment?

Reply:
I think the versions listed are "supported" in that they're known to build and work for the plugin, but not that they are "supported" upstream, obviously. IMO, if it's a simple matter of a few ugly `#ifdef` it's worthwhile even if just to allow people on slightly older distros to compile the latest plugin release tarball.

Reply:
Might be useful -- on Ubuntu's side, there's still one remaining release (trusty) that has webkitgtk-3.0 but not webkit2gtk-4.0.